extensions/fetch/23_request.js

```diff
@@ -273,7 +273,7 @@
         ((init.body !== undefined && init.body !== null) ||
           inputBody !== null)
       ) {
-        throw new TypeError("HEAD and GET requests may not have a body.");
+        throw new TypeError("HEAD or GET request cannot have a body.");
```

Comment:
Any thoughts on using chrome's error message: `Request with GET/HEAD method cannot have body`. I feel like it's more clear. But I'm okay with the above too.

Reply:
Ok let's use chrome's